sdk/communication/Azure.Communication.Common/tests/CommunicationTestEnvironment.cs

```diff
@@ -18,7 +18,7 @@ public class CommunicationTestEnvironment : TestEnvironment
 
         public string LiveTestDynamicConnectionString => GetRecordedVariable(
             LiveTestDynamicConnectionStringEnvironmentVariableName,
-            options => options.HasSecretConnectionStringParameter("accessKey", SanitizedValue.Base64));
+            options => options.IsSecret("endpoint=https://sanitized.communication.azure.com/;accesskey=Kg=="));
```

Comment:
Since `CommunicationTestEnvironment` is shared across all modalities, can you please run a couple of sister pipelines to make sure we didn't break it for others?

Reply:
@petrsvihlik I run pipelines for NetworkTraversal  and Rooms. They're green